addons/payment_razorpay/models/payment_transaction.py

```diff
@@ -20,6 +20,44 @@
 class PaymentTransaction(models.Model):
     _inherit = 'payment.transaction'
 
+    @api.model
+    def _validate_and_sanatize_phone_number(self, phone):
+        """ Sanatize and validate phone number if phone number is valid and if not
+        valid then raise validation error.
+        :param char phone
+        :return: Sanatized and validated phone number.
+        :rtype: char
+        """
+        phone = self.partner_phone
```

Comment:
So... You don't, in fact, use the phone parameter of the method?

Reply:
opps :astonished: